scripts/paths.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+let path = require('path');
+
+Object.defineProperty(exports, 'lib', {
+    get: function() {
+        let libPath = path.resolve(__dirname, '../inc/' + getLibraryName('napa'));
+        // Output the path to stdout for cmake/gyp commands.
+        process.stdout.write(libPath);
+        return libPath;
+    }
+});
+
+Object.defineProperty(exports, 'inc', {
+    get: function() {
+        let incPath = path.resolve(__dirname, '../inc');
+        // Output the path to stdout for cmake/gyp commands.
+        process.stdout.write(incPath);
+        return incPath;
+    }
+});
+
+Object.defineProperty(exports, 'third_party', {
```

Comment:
>s, 'third_party [](start = 28, length = 15)

Please don't export our third-party dir to users, it is not part of napa product.